doc/EXAM.md

```diff
@@ -0,0 +1,157 @@
+Electrical Exam Documentation
+=============================
+
+This documentation describes the functionalities added to the A+ system
+for electrical examination during Summer 2019 by a few summer workers.
+
+Table of contents
+-----------------
+
+* [Exammode app](#exammode-app)
+* [Exam Management Page](#exam-management-page)
+* [Selection of Exam Templates](#selection-of-exam-templates)
+
+Exammode app
+------------
+
+The exammode app is the bread and butter of the designed electrical exam system.
+
+### Models
+
+* ExamSession
+
+	`ExamSession` class is responsible for a single exam session.
+	`ExamSession` includes the details of a single exam.
+
+	* `course_instance`
+		The course for the exam.
+
+	* `exam_module`
+		The exam. The exam is currently a single A+ module.
+
+	* `room`
+		The physical room, where the exam is held.
+
+	* `active_exams`
```

Comment:
I thought that the `ExamSession.active_exams` is a model manager for ExamSessions, but here it says that it would link to the ExamAttempts.